scripts/aws/cloudformation/tasking-manager.template.js

```diff
@@ -169,9 +169,9 @@ const Resources = {
         'sudo apt-get -y install python3.6-dev',
         'sudo apt-get -y install python3.6-venv',
         'sudo apt-get -y install curl',
-        'curl -sL https://deb.nodesource.com/setup_6.x > install-node6.sh',
-        'sudo chmod +x install-node6.sh',
-        'sudo ./install-node6.sh',
+        'curl -sL https://deb.nodesource.com/setup_10.x > install-node10.sh',
```

Comment:
can we do `curl -o install-node10.sh -sL https://deb.nodesource.com/setup_10.x` . Better to use this than shell redirection.

Alternatively, to make this idempotent, we can use a temporary file, like so:
```
INSTALL=$(mktemp)
curl -o ${INSTALL} -sL https://deb.nodesource.com/setup_10.x
chmod a+x ${INSTALL}
sudo ${INSTALL}
```

Depending on the install situation, this may or may not help.

Reply:
Cool. Changed as suggested.